client/plots/matrix/matrix.cells.js

```diff
@@ -223,7 +223,18 @@ export function setGeneVariantCellProps(cell, tw, anno, value, s, t, self, width
 
 		// return the corresponding legend item data
 		const byDt = self.state.termdbConfig.assayAvailability?.byDt
-		const order = value.class == 'CNV_loss' ? -2 : value.class.startsWith('CNV_') ? -1 : 0
+		const order =
+			value.class == 'CNV_homozygous_deletion'
```

Comment:
please create a reusable helper to convert cnv class to value and eliminate code dup